lib/classes/CLI.js

```diff
@@ -65,8 +65,14 @@ class CLI {
 
   generateMainHelp() {
     this.consoleLog('');
+    this.consoleLog(chalk.yellow('The Serverless Framework - https://serverless.com'));
+    this.consoleLog('');
+
+    this.consoleLog(chalk.yellow('Version:'));
+    this.consoleLog(version);
+    this.consoleLog('');
 
-    this.consoleLog(chalk.yellow.underline('Commands'));
+    this.consoleLog(chalk.yellow('Commands:'));
```

Comment:
In my terminal, the underline is obnoxiously bold and as a result overwrites the bottom of the text.  Color works just fine to signal different sections.